routetable/route_table.go

```diff
@@ -122,6 +122,11 @@ func (r *RouteTable) SetRoutes(ifaceName string, targets []Target) {
 	r.dirtyIfaces.Add(ifaceName)
 }
 
+func (r *RouteTable) QueueResync() {
+	r.logCxt.Info("Queueing a resync.")
+	r.inSync = false
```

Comment:
How soon after this will Apply() be called?  It would be a shame if there was still a significant delay before a missing or superfluous route was corrected.